common/environment/setup/install.sh

```diff
@@ -34,17 +34,22 @@ _vsv() {
 	vmkdir etc/sv
 	vcopy "${FILESDIR}/$service" etc/sv
 	if [ ! -L $svdir/run ]; then
+		grep -q 'exec 2>&1' $svdir/run || msg_warn "$pkgver: vsv: service '$service' does not contain 'exec 2>&1' to log stderr\n"
 		chmod 755 $svdir/run
 	fi
 	if [ -e $svdir/finish ] && [ ! -L $svdir/finish ]; then
 		chmod 755 $svdir/finish
 	fi
 	ln ${LN_OPTS} /run/runit/supervise.${service} $svdir/supervise
-	if [ -d $svdir/log ]; then
-		ln ${LN_OPTS} /run/runit/supervise.${service}-log $svdir/log/supervise
-		if [ -e $svdir/log/run ] && [ ! -L $svdir/log/run ]; then
-			chmod 755 ${PKGDESTDIR}/etc/sv/${service}/log/run
-		fi
+	if [ -d $svdir/log ] || [ -L $svdir/log ]; then
+		msg_warn "$pkgver: vsv: overriding default log service\n"
+	else
+		mkdir $svdir/log
+		printf '#!/bin/sh\nexec vlogger -p daemon -t %s\n' "$service" > $svdir/log/run
```

Comment:
```suggestion
		cat <<-EOF  >$svdir/log/run
		#!/bin/sh
		exec vlogger -p daemon -t $service
		EOF
```

While the printf is correct, it isn't visualized as good as heredoc.